semgrep_output_v1.atd

```diff
@@ -1240,10 +1240,18 @@ type scan_info = {
     deployment_name: string;
 }
 
+type product_specific_ignores = {
+  ~sast: string list;
+  ~sca: string list;
+  ~secrets: string list;
+}
+
 (* config specific to the scan, eg *)
 type scan_configuration = {
     rules: raw_json; (* can we type this better *)
     inherit triage_ignored;
+    (* from 1.71.0 *)
```

Comment:
maybe better in engine_configuration next to ~ignored_files?

I'm also a bit confused why we have scan_config and scan_configuration ... this was part of some refactoring that Jonas started I think but we didn't finish it.
I'm not sure what goes in scan_configuration vs engine_configuration ... they are all useful to configure the engine.

Reply:
addressed in [58f73b3](https://github.com/semgrep/semgrep-interfaces/pull/248/commits/58f73b39e0d1a544892581f73a64a85e43ebaacf) and [8e11f3c](https://github.com/semgrep/semgrep-interfaces/pull/248/commits/8e11f3cc0bf43c5f8a0205c0a591627fb5e0bfa4)! 

I dropped `<json repr="object">` and that worked!